lib/src/kvs/tx.rs

```diff
@@ -1007,7 +1007,8 @@ impl Transaction {
 		}
 	}
 
-	pub(crate) fn clock(&self) -> Timestamp {
+	// Public for tests, but we might not want to expose this
+	pub fn clock(&self) -> Timestamp {
```

Comment:
Taking this function out of the crate is _technically_ a breaking change. Maybe not until we release, but it is easy to forget about

Reply:
Not breaking (subsequent invocations of the API are still the same). It is additive, and that is allowed.
The objection is that the clock is something we don't strictly want to expose.
I am adding a new clock here
https://github.com/surrealdb/surrealdb/pull/2510/files#diff-9ed97f4e3ea31932c4ac125955bd7389197d731e79b3bedef36be3f89d723207R1055
But that also seems to expose. Will tackle in the other PR and make a comment there now. Thanks for spotting!